.github/workflows/integration_tests.yml

```diff
@@ -433,7 +436,9 @@ jobs:
         timeout-minutes: 45
         continue-on-error: true
         env:
+          VERSION: ${{ env.VERSION }}
           KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
+          REMOTE_EXECUTION_PLANE: ${{ matrix.REMOTE_EXECUTION_PLANE }}
```

Comment:
This should not be needed, as it is already defined as an environment variable above.

Reply:
good catch - has been removed